features/promotion/managing_catalog_promotions/creating_catalog_promotion.feature

```diff
@@ -131,3 +131,11 @@ Feature: Creating a catalog promotion
         Then there should be 1 new catalog promotion on the list
         And it should have "winter_sale" code and "Winter sale" name
         And it should have priority equal to 10
+
+    @ui @javascript
```

Comment:
we could add `@no-api` tag

Reply:
I don't thinks so. API scenarios should have the `@api` tag and you either run Behat with `--tags="@api"` to include API scenarios or with `--tags="~@api"` to exclude them.

Reply:
`@no-api` is an already existing tag though. Perhaps it would make sense to remove it everywhere, if `~@api` is to be used instead.

Reply:
Then I'm curious why it exists...

Reply:
By the looks of it, Lukasz is the one who introduced the tag and it was added to about 80 scenarios, but I don't see it actually being used. Perhaps it's obsolete.  Let's see what Lukasz has to say in his defense.

Reply:
The `@no-api` tag was created to mark scenarios that we don't want to cover in API. The main reason was to be aware of how many features and scenarios still need to be covered

Reply:
Thanks for the clarification. It make a bit more sense now.
At this point, isn't it already easier to explicitly mark the scenarios that we want to cover in the API and assume that the rest are "@no-api"?
I'm just concerned that this is going to be confusing for users and contributors and probably even become redundant once all current features are covered in the new API.